html/cross-origin-opener-policy/META.yml

```diff
@@ -0,0 +1,4 @@
+spec: https://gist.github.com/annevk/6f2dd8c79c77123f39797f6bdac43f3e
+suggested_reviewers:
+  - mikewest
+  - jugglinmike
```

Comment:
I'd suggest adding @arturjanc (or @lweichselbaum).

Reply:
I added both and invited them to the [reviewers team](https://github.com/orgs/web-platform-tests/teams/reviewers). Artur & Lukas, welcome! You may want to [unwatch this repo](https://github.com/orgs/web-platform-tests/teams/reviewers/discussions/1), [filter emails](https://github.com/orgs/web-platform-tests/teams/reviewers/discussions/2) and check the [docs on reviewing tests](https://web-platform-tests.org/reviewing-tests/index.html). 🙂